src/azure-cli/azure/cli/command_modules/synapse/_params.py

```diff
@@ -128,6 +161,130 @@ def load_arguments(self, _):
+                   help='Type of client connection provider.',
+                   arg_type=get_enum_type(SqlPoolConnectionClientType))
+
+        auth_group = 'Authentication'
+
+        c.argument('auth_type',
+                   options_list=['--auth-type', '-a'],
+                   arg_group=auth_group,
+                   help='Type of authentication.',
+                   arg_type=get_enum_type(SqlPoolConnectionClientAuthenticationType))
+
+    # synapse sql pool classification
+    with self.argument_context('synapse sql pool classification') as c:
+        c.argument('sql_pool_name', arg_type=name_type, id_part='child_name_1', help='The SQL pool name.')
+
+    with self.argument_context('synapse sql pool classification list') as c:
+        c.argument('workspace_name', id_part=None, help='The workspace name.')
+
+    with self.argument_context('synapse sql pool classification recommendation list') as c:
+        c.argument('workspace_name', id_part=None, help='The workspace name.')
+        c.argument('include_disabled_recommendations', options_list=['--included-disabled'],
+                   arg_type=get_three_state_flag(),
+                   help='Indicates whether the result should include disabled recommendations')
+
+    for scope in ['show', 'create', 'update', 'delete', 'recommendation enable', 'recommendation disable']:
+        with self.argument_context('synapse sql pool classification ' + scope) as c:
+            c.argument('schema_name', help='The name of schema.', options_list=['--schema'])
+            c.argument('table_name', help='The name of table.', options_list=['--table'])
+            c.argument('column_name', help='The name of column.', options_list=['--column'])
+            c.argument('information_type', help='The information type.')
+            c.argument('label_name', help='The label name.', options_list=['--label'])
+
+    # synapse sql pool tde
+    with self.argument_context('synapse sql pool tde') as c:
+        c.argument('sql_pool_name', arg_type=name_type, id_part='child_name_1', help='The SQL pool name.')
+        c.argument('status', arg_type=get_enum_type(TransparentDataEncryptionStatus),
+                   required=True, help='Status of the transparent data encryption.')
+
+    # synapse sql pool threat-policy
+    with self.argument_context('synapse sql pool threat-policy') as c:
+        c.argument('sql_pool_name', arg_type=name_type, id_part='child_name_1', help='The SQL pool name.')
+
+    with self.argument_context('synapse sql pool threat-policy update') as c:
+        _configure_security_or_audit_policy_storage_params(c)
+
+        policy_arg_group = 'Policy'
+        notification_arg_group = 'Notification'
+
+        c.argument('state',
+                   arg_group=policy_arg_group,
+                   help='Threat detection policy state',
+                   arg_type=get_enum_type(SecurityAlertPolicyState))
+
+        c.argument('retention_days',
```

Comment:
Remove blank line between `c.argument` may be better. 
Not an important issue, it depends on you.

Reply:
fixed

---
In reply to: [533267134](https://github.com/Azure/azure-cli/pull/16095#discussion_r533267134) [](ancestors = 533267134)